### ISSUE TITLE
Import and export data via CSV within the GUI (without any developer experience needed)

### ISSUE THREAD

**Issue by toddpadwick** [sanity-io/sanity#4660]
### The problem
I am aware there are ways around this by using the API. However, this is only possible for a developer to set up. Our clients often wish to import and export data via CSV – a common feature of many other CMS's. 

A good example for this, is our 'redirects' doc type. These are for all the redirect entries from a previous site. This is often set up in a CSV, however, the client has to manually enter them one by one – all 500 of them!

Sanity is fantastic. But, this is one of many shortfalls of Sanity, where the focus has been placed on the developer experience over that of the end user – who likely won't have any technical know how.  In my view, both should be catered to in equal measure - As a developer, it is, after all, the end user who pays my bills ;)

### Solution
The ability to:
**Import via GUI**
A tool which allows a user to upload a CSV, and then match data from each column to a field type in a specific doc type.

**Export via GUI**
a tool which allows a user to export all documents created for one or more types, to a CSV.

**Describe alternatives you've considered**
There are documented methods to import and export data, but all of them require a developer, and cannot be achieved via a GUI.

**Comment by damienmutt** [comment]
Anyone has links to those "workarounds"? Because I can't figure out how to export data using NextJS, the only JS related doc is this _README_ which does not help: https://github.com/sanity-io/sanity/tree/next/packages/%40sanity/export

**Comment by toddpadwick** [comment]
No I havn't. This is becoming really frustrating for me, too. One of my clients has a team of SEO experts in place, none of which are developers – but they expected to be able to freely import and export their redirect CSV files which cannot be done with Sanity. They were pretty shocked at this considering they've had that ability with every other CMS they've worked with. Not a good look to be honest.

**Comment by damienmutt** [comment]
I end up "_exporting_" data manually querying Sanity documents, then saving result into a JS `const` and finally using a third party JSON to CSV lib ([export-from-json](https://github.com/zheeeng/export-from-json))
Not ideal, not fancy but... I had no time.

I love Sanity but a GUI way to import/export is a basic for any CMS.

**Comment by toddpadwick** [comment]
I agree @damienmutt - sanity is fantastic. But I do feel they forget that the end users (our clients who are not technical) are the ones who foot the bill. Whereas Sanity puts developer experience over their experience.

**Comment by kelvinyelyen** [comment]
Hello, any workaround for this issue?

**Comment by hartwm** [comment]
This is a big one for me in making the switch. I think @toddpadwick is spot on. The users who foot the bill are not technical and not having this easy of import or ability to bulk edit is something that is core to all CMS. I'm trying to switch our agency to sanity, but this is one thing holding us back.

**Comment by toddpadwick** [comment]
I've had another request for this feature in the CMS, to bulk import content via CSV. my client is a media publishing business. six times a year, they publish 'annual lists' covering a wide range of topics. These lists are all created in CSVs, initially, as they get published in both print and online. For online, they wish to import them automtically, resulting in 200 or so documents created from each row. They do not want to require developers to do this each time. @bjoerge I notice you opened one of my other [bulk content management tickets](https://github.com/sanity-io/sanity/issues/4971) a few months back, so I wonder if you had any feedback on this on? Appreciate your consideration.
